### PR TITLE
Slurm script with hparams for the 181B model

### PR DESCRIPTION
**Pull request by ibeltagy**
Please check the TODOs in the script.

### CODE REVIEW

File: train/tr11-200B-ml/tr11-200B-ml.slurm [hunk 1]
```diff
@@ -53,34 +53,41 @@ export TRANSFORMERS_OFFLINE=1
...
 
-PP_SIZE=10
+PP_SIZE=6
 TP_SIZE=8
 
-MICRO_BATCH_SIZE=1
-GLOBAL_BATCH_SIZE=2048
-TRAIN_SAMPLES=195_312_500 # 400B tokens
+MICRO_BATCH_SIZE=4
+GLOBAL_BATCH_SIZE=1568  # 3.21M tokens. TODO: check this number doesn't break constraints
```
Comment (stas00) [inline train/tr11-200B-ml/tr11-200B-ml.slurm:64]: why not 2048? we haven't been benchmarking 3/4 of 2048 - potentially will make the pipeline less efficient -> slower throughput.

discuss on slack?

File: train/tr11-200B-ml/tr11-200B-ml.slurm [hunk 2]
```diff
@@ -2,12 +2,12 @@
 #SBATCH --job-name=tr11-200B-ml
 #SBATCH --partition=gpu_p5
 #SBATCH --constraint=a100
-#SBATCH --nodes=10
+#SBATCH --nodes=10                   # TODO: update to 48
```
Comment (stas00) [inline train/tr11-200B-ml/tr11-200B-ml.slurm:5]:
```suggestion
#SBATCH --nodes=48
```

**Review by stas00 — APPROVED** [review]
Looks great, thank you Iz

Just not sure about GBS - let's discuss on slack

File: train/tr11-200B-ml/tr11-200B-ml.slurm (hunk 1 above)
Comment (ibeltagy) [reply train/tr11-200B-ml/tr11-200B-ml.slurm:64]: ok. It is probably fine to use GBS=2048 but we will need to increase `--rampup-batch-size` a bit. I will post on slack tomorrow morning.

Comment (stas00) [reply train/tr11-200B-ml/tr11-200B-ml.slurm:64]: we can also benchmark this new suggestion and then decide. I'm out of nodes right now, so can do tomorrow.

**Pull request merged**